org.kde.krita.yaml

```diff
@@ -17,6 +15,7 @@ finish-args:
   - --filesystem=xdg-run/gvfs
   - --env=PYTHONPATH=/app/lib/python3/dist-packages
   - --env=TMPDIR=/var/tmp
+  - --env=XDG_CURRENT_DESKTOP=gtk3
```

Comment:
Does this works on KDE desktops without GTK portals?

Reply:
Yes, the Krita directory selector doesn't use portals so it brings up a regular GTK file picker.